impl/config/config.go

```diff
@@ -56,7 +56,8 @@ type DHTServiceConfig struct {
 }
 
 type PKARRServiceConfig struct {
-	RepublishCRON string `toml:"republish_cron"`
+	RepublishCRON   string `toml:"republish_cron"`
+	CacheTTLMinutes int64  `toml:"cache_ttl_minutes"`
```

Comment:
Consider making this a duration, so users can specify resolutions finer than per-minute.

Reply:
Good idea